FluidNC/src/Motors/NullMotor.h

```diff
@@ -2,15 +2,43 @@
 
 #include "MotorDriver.h"
 
+#include <cstdint>
+
 namespace MotorDrivers {
     class Nullmotor : public MotorDriver {
+        int32_t direction = 0;
+        int32_t steps     = 0;
+        int     state     = 0;
```

Comment:
Is state actually used anywhere?  It gets set and unset but other than that I don't see anywhere that the information goes.

Reply:
Might be a left-over. I have to check. Let's first decide what to do with the PR.